person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] Kindly assign `@id` with a valid value as the `id` doesn't exist. You can use the following:
```ruby
@id = Random.rand(1..1000)
```